rllib/algorithms/dqn/torch/dqn_rainbow_torch_rl_module.py

```diff
@@ -37,17 +37,18 @@ def setup(self):
         # If we use a noisy encoder. Note, only if the observation
         # space is a flat space we can use a noisy encoder.
         self.uses_noisy_encoder = isinstance(self.encoder, TorchNoisyMLPEncoder)
-        # We do not want to train the target networks.
-        # AND sync all target nets with the actual (trained) ones.
-        self.target_encoder.requires_grad_(False)
-        self.target_encoder.load_state_dict(self.encoder.state_dict())
+        if self.is_learner_module:
```

Comment:
Nice!

Reply:
Nit: Should we use super-constrained naming here and either always use either ...?
1) "light"
2) "learner"
but not both terms mixed.

I like the term "light". Or "inference_only".

So - if we chose one of these - we would change the if block to:
```
if not self.is_light_module:
    ...
```